ga4gh/client.py

```diff
@@ -278,6 +296,35 @@ def searchVariantAnnotations(
             request, "variantannotations",
             protocol.SearchVariantAnnotationsResponse)
 
+    def searchFeatures(
+            self, featureSetId=None, parentId=None, referenceName=None,
+            start=0, end=None, featureTypes=[]
+                       ):
+        """
+        Returns the result of running a searchFeatures method
+        on a request with the passed-in parameters.
+        :param featureSetId: string ID of the feature Set being searched
+        :param parentId: string ID (optional) of the parent feature
+        :param referenceName: string name of the reference to search
+            (ex: "chr1")
+        :param start: int search start position on reference
```

Comment:
colon and types are in the wrong place here and elsewhere, should be:
```
:param int start: search start position on reference
```
Please check these in other sphinx docs.